src/anime_graph.cpp

```diff
@@ -90,24 +126,46 @@ Node* AnimeGraph::getNode(std::string anime_name) const {
     return NULL;
 }
 
-// Returns the node associated with the specified anime_name. If it does not exist, return NULL
+/**
+ * @brief finds the pointer to the anime node with the given id
+ * 
+ * @param anime_id 
+ * @return Node* 
+ */
 Node* AnimeGraph::getNode(unsigned anime_id) const {
     if (!node_list.contains(anime_id)) return NULL;
 
     return node_list.at(anime_id);
 }
 
-// Returns a constant pointer to the KDTree
+/**
+ * @brief return a constatn pointer to the KDTree
+ * 
+ * @return KDTree* 
+ */
 KDTree* AnimeGraph::getTree() const { return tree; }
 
 /* Check existence of node or edge */
 
-// Returns whether the node exists in adjacency_list
+/**
+ * @brief return whether a node with the given id exists in node_list
+ * 
+ * @param anime_id 
+ * @return true 
+ * @return false 
```

Comment:
Only one is needed, say something like: returns true if the node exists, false otherwise